sdk/src/types/block/output/unlock_condition/expiration.rs

```diff
@@ -55,6 +60,12 @@ impl ExpirationUnlockCondition {
     }
 }
 
+impl StorageScore for ExpirationUnlockCondition {
+    fn storage_score(&self, params: StorageScoreParameters) -> u64 {
+        self.return_address().storage_score(params)
```

Comment:
What about the expiration slot index?
Guess it doesn't really matter because it's just 0, but same would be true for the address?

Reply:
Same about the amount field in the SDRUC

Reply:
I don't know if TIPs disallow that but you could have an implicit account creation address in the address and it then comes with an offset. Slot index has no chance to incur any additional cost.

Reply:
![image](https://github.com/iotaledger/iota-sdk/assets/46689931/c6d8c339-d43d-42e5-9782-f057ddb273c5)
Implicit account creation address is not listed here

Reply:
Fair enough. But tbh I think I would keep it like that, a bit less error prone if we ever add an address type that comes with an offset.